reference/conanfile/tools/gnu/pkgconfigdeps.rst

```diff
@@ -92,4 +92,4 @@ flags and require relations.
 
 Additionally, a *<PKG-NAME>.pc* is generated to maintain compatibility for consumers with recipes that start supporting components. This
 *<PKG-NAME>.pc* file will declare all the components of the package as requires while the rest of the fields will be empty, relying on
-the propagation of flags coming from the components *<COMP-NAME>.pc* files.
+the propagation of flags coming from the components *<[PKG-NAME]-[COMP-NAME]>.pc* files.
```

Comment:
This requires changing also the above reference to *<COMP-NAME>.pc* file

Reply:
Absolutely 😁